go/vt/vtgr/controller/refresh_test.go

```diff
@@ -0,0 +1,145 @@
+/*
+Copyright 2021 The Vitess Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controller
+
+import (
+	"context"
+	"fmt"
+	"sort"
+	"testing"
+	"time"
+
+	"vitess.io/vitess/go/vt/vtctl/grpcvtctldserver/testutil"
+
+	"vitess.io/vitess/go/vt/topo/memorytopo"
+
+	"vitess.io/vitess/go/vt/vtgr/config"
+
+	"vitess.io/vitess/go/vt/logutil"
+
+	"github.com/stretchr/testify/assert"
+
+	topodatapb "vitess.io/vitess/go/vt/proto/topodata"
+
+	"vitess.io/vitess/go/vt/topo"
+
+	"github.com/golang/mock/gomock"
+)
```

Comment:
Why is this formatted like this? Can you run `goimports` on all the new files?
I'm concerned that this may cause problems for someone who does have the pre-commit hooks installed.
As an aside we probably need a goimports CI check.

Reply:
I am not sure.. goimports seems to be happy with it... anyway i fixed the order here